src/main/java/com/minecolonies/coremod/colony/Structures.java

```diff
@@ -627,15 +852,21 @@ else if (name.startsWith(SCHEMATICS_CACHE + SCHEMATICS_SEPARATOR))
         }
 
         /**
-         * Whether the schematic is a huit or not.
-         * This is done using the naming convention only, should start by huts/
-         * and a minecolonies block name should exist.
+         * Create a StructureName
          *
-         * @return True is it is a hut otherwise False
+         * @param prefix    should be schematics, scan or cache.
+         * @param style     ex: wood, stone, walls/stone
+         * @param schematic as in Builder1, Gate, without the nbt extension.
          */
-        public boolean isHut()
+        public StructureName(@NotNull final String prefix, final String style, @NotNull final String schematic)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this constructor to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1213)